path_roles.go

```diff
@@ -31,6 +31,13 @@ type roleEntry struct {
 	TTL                 time.Duration `json:"ttl"`
 	MaxTTL              time.Duration `json:"max_ttl"`
 	PermanentlyDelete   bool          `json:"permanently_delete"`
+	PersistApp          bool          `json:"persist_app"`
+
+	// Info for persisted apps
+	RaIDs                      []string `json:"role_assignment_ids"`
```

Comment:
Missed this one in the last review, sorry!

```suggestion
	RoleAssignmentIDs                      []string `json:"role_assignment_ids"`
```